scripts/make_slack_notification_message.py

```diff
@@ -122,16 +122,10 @@ def section_block(text: str) -> dict:
 
     print(
         json.dumps(
-            {
-                "attachments": [
-                    {
-                        "blocks": [header_block("Archiver Run Outcomes")]
-                        + failed_blocks
-                        + changed_blocks
-                        + unchanged_blocks,
-                    }
-                ]
-            },
+            [header_block("Archiver Run Outcomes")]
+            + failed_blocks
+            + changed_blocks
+            + unchanged_blocks,
```

Comment:
An array of blocks, as required for the `blocks` input to the [`chat.postMessage()` method](https://api.slack.com/methods/chat.postMessage).